<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyGUIWithCreator</class>
 <widget class="QWidget" name="MyGUIWithCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyGUIWithCreator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="dogList"/>
     </item>
     <item>
      <widget class="QPushButton" name="addAdoptionButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="adoptionList"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
